adding_species_manuscript.tex

```diff
@@ -426,9 +429,10 @@ \section*{Additions to \stdpopsim}
 For example, the model for \emph{Drosophila melanogaster} has been updated in the \stdpopsim catalog to have a fraction of gene conversions of 0.83 (in all chromosomes with recombination) and an average tract length of 518 bases \citep{Comeron2012}.
 We note, however, that since non-crossover recombination incurs a high computation load in simulation,
 it is turned off by default in \stdpopsim, and must be explicitly invoked by the simulation model.
+Note that unless this option is explicitly turned on in simulation, the effective fraction of gene conversions is 0, even if a positive fraction is specified for that species.
 
```

Comment:
I added this to clarify what happens if you specify a non-0 fraction of GC but do not explicitly turn on the GC flag. This is a bit technical, but potentially important. Pinging @petrelharp to check that I got it right.

Reply:
Gee, isn't this what the previous sentence says? Maybe an even-more-clear and less redundant clarification than the sentence you wrote would be
```
We note, however, that since non-crossover recombination incurs a high computation load in simulation,
it is turned off by default in \stdpopsim, and must be explicitly invoked by the simulation model,
even for species like \textit{Drosohpila melanogaster} that have gene conversion parameters specified.
```

Reply:
yes, this is better. Will change in next commit

Reply:
@dschride - to answer your question, if we do not turn on the "GC flag" when simulating a species that has GC in its model (fraction of GC >0), the rate of recombination will be lower. We're just not simulating the GC events. I'd rather not get into these details in the manuscript. Just clarify that specifying a GC fraction > 0 doesn't automatically turn the GC flag on.

Reply:
This is not right, @igronau - the GC flag does not affect the rate of crossing-over, and the rate of crossing-over is always equal to the specified recombination rate. I *think* this is what [the docs say](https://popsim-consortium.github.io/stdpopsim-docs/latest/api.html#gene-conversion-and-bacterial-recombination) but if someone has suggestions for how to make it more clear, please let us know.

Reply:
Hm, unless you meant that the "effective rate of recombination" will  be lower in the same sense I wrote in the docs? There I said that
> enabling gene conversion effectively increases the rate of recombination, which can have a strong effect on runtimes
because gene conversion adds more locations where the trees change, like crossing-over does.

Reply:
I might actually have gotten this wrong. I think I now understand this, and maybe this does require clarification in the text. My new suggestion is to clarify this using the Drosophila example. I pasted the text below, with my addition in bold
> For example, the model for Drosophila melanogaster has been updated in the stdpopsim catalog to have a fraction of gene conversions of 0.83 (in all chromosomes that undergo recombination) and an average tract length of 518 bases (Comeron et al., 2012). **This update does not affect the rate of crossover recombination, but it adds gene conversion events at a rate of 83:17 (roughly 5:1) relative to crossover recombination events.** We note that since non-crossover recombination incurs a high computation load in simulation, it is turned off by default in stdpopsim, and must be explicitly invoked by the simulation model. 

I think that this clarifies where the additional computation load comes from and also the fact that if you don't turn on the flag, you'll be modeling non-CO recombination at its correct rates, but you'll be ignoring the many GC events. What do you say, @dschride @petrelharp ?

Reply:
See text in latest commit and let me know what you think

Reply:
Yes I think that is perfectly clear.

Reply:
I removed this in my latest commit